### PR TITLE
Upgraded Electron and its deps to latest stable

### PR DESCRIPTION
**Pull request by spaquet**

### CODE REVIEW

File: app/features/config/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default {
     /**
      * Application name.
      */
-    appName: 'Jitsi Meet',
+    appName: 'Tocca Virtual Event',
```
Comment (0x46616c6b) [inline app/features/config/index.js]: I guess this shouldn't be part of this PR, right?

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "jitsi-meet-electron",
-  "version": "1.1.1",
-  "description": "Electron application for Jitsi Meet",
+  "version": "1.1.2",
+  "description": "Electron application for Tocca Virtual Event",
```
Comment (0x46616c6b) [inline package.json]: Also here a lot of changes regarding the branding shouldn't be part of this PR.

**Comment by saghul** [comment]
@spaquet This PR includes much more than dependency updates. Any chance you could isolate those?

File: package.json [hunk 3]
```diff
@@ -66,9 +66,9 @@
     "url": "git://github.com/spaquet/jitsi-meet-electron"
   },
   "keywords": [
-    "jingle",
-    "webrtc",
-    "xmpp",
+    "conference",
```
Comment (saghul) [inline package.json]: please drop this change

File: main.js [hunk 4]
```diff
@@ -22,6 +22,7 @@ const config = require('./app/features/config');
 
 // We need this because of https://github.com/electron/electron/issues/18214
 app.commandLine.appendSwitch('disable-site-isolation-trials');
+app.allowRendererProcessReuse = false; // (change to true in Electron 9)
```
Comment (saghul) [inline main.js]: why is this necessary?

**Comment by saghul** [comment]
Supersededd by https://github.com/jitsi/jitsi-meet-electron/pull/227
